src/www/ui/search-helper.php

```diff
@@ -36,8 +36,8 @@
  *         contains uploadtree_pk, parent, upload_fk, pfile_fk, ufile_mode, and
  *         ufile_name
  */
-function GetResults($Item, $Filename, $Upload, $tag, $Page, $SizeMin, $SizeMax, $searchtype,
-                    $License, $Copyright, $uploadDao, $groupID, $PG_CONN, $limit = 100)
+function GetResults($Item, $Filename, $Upload, $tag, $Page = 1, $SizeMin, $SizeMax, $searchtype,
```

Comment:
There can be a parameter with default value before a parameter without default value.

Also, do we really need default value for `$Page`?

Reply:
Yes I have seen that i have anyhow set it it to default 1 here https://github.com/Shruti3004/FOSSology-REST-API/blob/d47de4c7103d7ae29e8fc86afe46df2531b85fef/src/www/ui/api/Controllers/SearchController.php#L85 so i have removed it